.env.example

```diff
@@ -7,8 +7,16 @@ NET_VERSION=171022
 ACCOUNT_MANAGER_SECRET=
 # Should be the externally facing IP of your server host
 EXTERNAL_IP=localhost
+
+# The database type that will be used.
+# Acceptable values are `sqlite`, `mysql`, `mariadb`, `maria`.
+# Case insensitive.
+database_type=mariadb
```

Comment:
Should be uppercase, we transform that on lookup

Reply:
ah yeah, copy paste error